.github/CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# More details are here: https://help.github.com/articles/about-codeowners/
+
+# The '*' pattern is global owners.
+
+# Order is important. The last matching pattern has the most precedence.
+# The folders are ordered as follows:
+
+# In each subsection folders are ordered first by depth, then alphabetically.
+# This should make it easy to add new rules without breaking existing ones.
+
+* @grafana/aws-datasources
+* @grafana/oss-big-tent
+* @grafana/partner-datasources
+* @grafana/enterprise-datasources
+* @grafana/plugins-platform-backend
```

Comment:
Do we need plugins platform?

Reply:
Sorry it is, my question was if we should remove them. I'm not sure they really own this repo